### ISSUE TITLE
Autofill issue with Firefox 83.1 (Android 9/10)

### ISSUE THREAD

**Issue by daviddem1971** [bitwarden/mobile#1167]
## Describe the Bug

It seems enabling accessibility in BW interferes with autofill in Firefox.
## Steps To Reproduce

1. in BW autofill services, enable autofill, disable accessibility
2. open Google sign-in in Firefox (or any other site with matching logins in your vault)
3. as expected, obtain a BW pop up with possible matches from the vault
4. in BW autofill, enable autofill, enable accessibility, enable draw over
5. open Google sign-in in Firefox
6. now the possible matches from the vault do not show up. Only the "go to my vault" option:
## Expected Result

Expected that Firefox would favor autofill over accessibility, as do other browsers and apps
## Actual Result

Autofill does not work in Firefox when accessibility is enabled in BW
## Screenshots or Videos

Firefox with autofill enabled and accessibility disabled in BW:

![Screenshot 1](https://user-images.githubusercontent.com/47311661/101362327-421c3600-38a0-11eb-8c49-0994b0dc26d8.png)

Firefox with autofill, accessibility and draw over enabled in BW:

![Screenshot 2](https://user-images.githubusercontent.com/47311661/101362463-6c6df380-38a0-11eb-8e12-81fb871ac11e.png)
## Environment

- Device: Samsung S9+ / Samsung A20e
- Operating system: Android 9 / Android 10
- Firefox 83.1 or Firefox 84 beta
- BW app 2.7.0 (3258)

## Additional Context

I did try the above with the BW extension for Firefox enabled or disabled, still same result.

**Comment by SergeantConfused** [comment]
Hi there,

Thank you for bringing this to our attention.
Do you happen to have a Samsung device and have Bitwarden installed in the Secure Folder too?


Thank you in advance,

**Comment by daviddem1971** [comment]
> 
> 
> Hi there,
> 
> Thank you for bringing this to our attention.
> Do you happen to have a Samsung device and have Bitwarden installed in the Secure Folder too?
> 
> Thank you in advance,

It is a rooted Samsung S9+ with a custom ROM without secure folder. Normal BW installation from Playstore.

I do have another non-rooted Samsung device with Android 10 if you need me to do some testing.

**Comment by daviddem1971** [comment]
Tested with my non-rooted phone, basically new out of the box, Samsung A20e, Android 10, fresh installed Firefox and Bitwarden: same issue.

**Comment by LaXiS96** [comment]
Same issue here, but I may have "fixed" it. On a OnePlus 6 running both Firefox 84.1.4 and Firefox Nightly 210124 17:01.

Had both app and Firefox extension installed, with autofill service enabled, accessibility disabled. Removing the Firefox extension made no difference. What helped was simply disabling and reenabling the autofill service, after that I haven't seen the second popup again.

**Comment by eliykat** [comment]
Thanks for reporting this issue. Autofill problems can affect different sites, apps, or devices, and we’re working on improving this feature. To help us track and analyze affected sites, please lodge a report using the Google Form mentioned in this issue: #1389. Please also direct any discussion or questions to that issue. This issue will now be closed.
